code/modules/projectiles/projectile/energy.dm

```diff
@@ -136,7 +136,15 @@
 
 /obj/item/projectile/energy/toxplasma
```

Comment:
Might want to change the name to involve actual burn damage instead of tox damage

Reply:
Waiting on maints, as it is a map change

Reply:
Done